toolchain/templates/default.mako

```diff
@@ -33,28 +33,28 @@ warn "Consider using a different template via the $MAGENTA--computer$COLOR_RESET
     ${helpers.run_prologue(target)}
 
     % if not mpi:
-        (set -x; ${' '.join([f"'{x}'" for x in profiler ])} "${target.get_install_binpath(case)}")
+        (${profiler} "${target.get_install_binpath(case)}")
```

Comment:
I think you also forgot to add back `set -x` here.

Reply:
oopsies. Fixed and did a more careful check on my own before pushing this time.